model/clusters_mgmt/v1/events_resource.model

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages the collection of events.
+resource Events {
+	// Retrieves the list of events.
+	method List {
+		// Retrieved list of events.
+		out Items []Event
```

Comment:
As events don't have identifiers, and there is no paging or search capability here I assume that this will return always all the events. Or will the set of events restricted by user? Please document how this works.

Reply:
Removed